### PR TITLE
Created simple example (closes #3)

### LINKED ISSUE #3: Create examples/simple

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

File: examples/simple/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <ul>
+      <section
+        ><h2> </h2
+        ><div class="background"
+          ></div
+          ></section>
+      <li
+        ><h3> </h3
+        ><p> </p
+        ><div class="overlay"
```
Comment (etiennesegonzac) [inline examples/simple/index.html]: We can remove L21 and L22 since we have no edit mode

File: examples/simple/app.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/*global DataSource, ScheduledList, scheduler*/
+
+var content = document.getElementById('content');
+var data = createDummyData(100);
```
Comment (etiennesegonzac) [inline examples/simple/app.js]: to 1000 :)

File: examples/simple/app.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+              linear-gradient(to bottom, transparent 0px, transparent 15px, #EDEDED 15px, #EDEDED 25px, transparent 20px),
+              linear-gradient(to top, #bcbcbc, #bcbcbc 1px, white 1px, white);
+  background-position: 20% 88px, 5% -88px, 0 -88px;
+  background-size: 90% 88px, 60% 88px, 100% 88px;
+  background-repeat: repeat-y;
+
+  z-index: 0;
+}
+
+ul.reordering .background {
```
Comment (etiennesegonzac) [inline examples/simple/app.css]: we can remove this one

File: examples/simple/app.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  width: 100%;
+
+  background: linear-gradient(to bottom, transparent 0px, transparent 45px, #EDEDED 45px, #EDEDED 70px, transparent 20px),
+              linear-gradient(to bottom, transparent 0px, transparent 15px, #EDEDED 15px, #EDEDED 25px, transparent 20px),
+              linear-gradient(to top, #bcbcbc, #bcbcbc 1px, white 1px, white);
+  background-position: 20% 88px, 5% -88px, 0 -88px;
+  background-size: 90% 88px, 60% 88px, 100% 88px;
+  background-repeat: repeat-y;
+
+  z-index: 0;
```
Comment (etiennesegonzac) [inline examples/simple/app.css:44]: maybe we can keep just the "line" part of the gradient (last background line, last position, last size (I think))

**Comment by wilsonpage** [comment]
@etiennesegonzac can you take a look at why we have one blank section header at the top?

**Comment by etiennesegonzac** [comment]
Sure !

**Comment by etiennesegonzac** [comment]
so because of #21 the fix is something like
(also simplifying the background a bit since it's a simple example :))

```
diff --git a/examples/simple/app.css b/examples/simple/app.css
index 689877a..a5028c9 100644
--- a/examples/simple/app.css
+++ b/examples/simple/app.css
@@ -31,43 +31,26 @@ ul {
 }

 ul section {
   margin: 0;
   padding: 0;
 }

 ul h2 {
-  position: sticky;
-  position: -webkit-sticky;
-  top: 0px;
-  height: 32px;
-
-  margin: 0;
-  padding: 0 6px;
-  box-sizing: border-box;
-
-  font-size: 0.9em;
-  line-height: 32px;
-
-  background: #efefef;
-  border-bottom: 1px #bcbcbc solid;
-
-  z-index: 100;
+  display: none;
 }

 .background {
   position: absolute;
   width: 100%;

-  background: linear-gradient(to bottom, transparent 0px, transparent 45px, #EDEDED 45px, #EDEDED 70px, transparent 20px),
-              linear-gradient(to bottom, transparent 0px, transparent 15px, #EDEDED 15px, #EDEDED 25px, transparent 20px),
-              linear-gradient(to top, #bcbcbc, #bcbcbc 1px, white 1px, white);
-  background-position: 20% 88px, 5% -88px, 0 -88px;
-  background-size: 90% 88px, 60% 88px, 100% 88px;
+  background: linear-gradient(to top, #bcbcbc, #bcbcbc 1px, white 1px, white);
+  background-position: 0 -88px;
+  background-size: 100% 88px;
   background-repeat: repeat-y;

   z-index: 0;
 }


 ul li {
   width: 100%;

```

**Comment by etiennesegonzac** [comment]
I'l like to add a small manifest.webapp too since the WebIDE is still the fastest way to iterate on device.

```
{
  "name": "simple-list-demo",
  "description": "Sweet",
  "launch_path": "/index.html",
  "icons": {
    "16": "/icons/icon16x16.png",
    "48": "/icons/icon48x48.png",
    "60": "/icons/icon60x60.png",
    "128": "/icons/icon128x128.png"
  },
  "orientation": "default",
  "permissions": {}
}

```

**Comment by wilsonpage** [comment]
@etiennesegonzac done

**Comment by etiennesegonzac** [comment]
\o/

**Pull request merged**